src/customer.spec.ts

```diff
@@ -229,14 +234,13 @@ describe("querier", () => {
       },
     };
     const customer = newCustomer(hooks);
-    const spyPaginatedSearch = mockPaginatedSearch(customer);
+    mockPaginatedSearch(customer);
     mockParse(mockQueryReturnValue);
     mockGetGoogleAdsError(customer);
     const spyHook = jest.spyOn(hooks, "onQueryError");
     // @ts-expect-error private method
     await customer.querier(mockGaqlQuery);
 
-    expect(spyPaginatedSearch).not.toThrow();
```

Comment:
The line above (242) will throw if `paginatedSearch` throws, making the test fail. So this line is redundant, and will never fail the assertion.

The other reason that I removed it is that I couldn't get the darn thing to work. For some reason including this single `expect` makes the test call the mocked `parse` function twice instead of just once, and one of the calls is missing some required arguments. I don't know why. There's something funky about the way we're doing spies, but that's outside the scope of this PR imo